data/person.json

```diff
@@ -0,0 +1,15 @@
+[
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/225153610-c08fac5a-9f5c-43d4-b71b-53b17700ffb5.png)

- [ ] Great work on preserving the data into files so far :blush: it would be nice if you handle that case where one or more JSON files are missing from your code :thinking: You can see in the image above I deleted one JSON file and started the application and your application showed me this error :fearful: Kindly fix this error :+1:

**Hint: You can check on the startup of the application, if a file doesn't exist then create one file with its respective name :+1:** :point_down:

```ruby
    file_path = "./data/#{file}.json"
    File.new(file_path, 'w+') unless File.exist?(file_path)
```